client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,18 +44,25 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Support for Camunda Cloud 1.3</h4>
-          With this release, we added support for Camunda Cloud 1.3.<br />
-          You can now assign a User Task to a specific assignee or to a group of candidates.
-          Both fields can be configured in the assignment section of the properties panel using a static value or an expression.
-          Learn more about this feature in the <a href="https://docs.camunda.io/docs/reference/bpmn-processes/user-tasks/user-tasks/#assignments">Camunda Cloud docs.</a>
+          <h4>Support for Camunda Cloud 1.4</h4>
+          With this release, we added support for Camunda Cloud 1.4.<br />
+          You can now evaluate DMN decision in a Business Rules Task.
+          Specify called decision by ID and assign the result to a variable via the properties panel.
```

Comment:
Hmmm. Maybe a bit clearer: 

```suggestion
          You can now evaluate a DMN decision in a Business Rules Task.
          Specify the called decision (DMN) by its ID and assign the result to a variable via the properties panel.
```

WDYT?